bots/log2zulip

```diff
@@ -9,6 +9,7 @@ import sys
 import shutil
 import subprocess
 import traceback
+from typing import List
```

Comment:
This should be after the last `sys.path.insert` below, so that this script doesn't need `typing` installed globally on the server.